src/config.ts

```diff
@@ -0,0 +1,22 @@
+import { getInput } from '@actions/core';
+import * as rt from 'runtypes';
+
+export const command = rt.Union(
```

Comment:
@stack72 Should we restrict to only allow these or more?

Reply:
we also need to add support for preview

Reply:
Great, adding!

Reply:
Fixed! :)